### ISSUE TITLE
WPML Theme Options Issue

### ISSUE THREAD

**Issue by ghostpool** [reduxframework/redux-framework#2388]
Support URL: http://support.redux.io/?id=gtaZ39RVXs1KznUYFkt%2BxDGWsx6qwOoj8WQJytUyfrb1z7X5gIgvaYqwj5X8UnXarWz08wEiPEwCMB9oS32Xdw%3D%3D

There appears to be a couple of issues when using WPML on the Redux theme options page:

1) Fields remain the same across each language. For example if I change a text field under English then switch to French, the English text is still there. If I change the text to French and switch back to English it now has the French text. This isn't a major issue as I can translate the theme options through WPML's own option as I've added the option names to wpml-config.xml file.

2) The more import issue, which I cannot resolve, is that I have a multi select field that displays categories. Under English it only displays the English categories, under French is only displays the French categories as expected. However when I select the English categories it removes the French categories and vice versa. There's no way to have the English and French categories both selected under their respective languages.

I'm not sure if this is a Redux or WPML issue but I wanted to mention it so you could look into it as WPML does not seem to be working at all within the Redux theme options page.

**Comment by kprovance** [comment]
Have you reviewed this?  https://docs.reduxframework.com/core/advanced/wpml-integration/  I suspect anything further will require more R&D, which means we'd need financial support, via ideas.reduxframework.com

**Comment by ghostpool** [comment]
Yes I have already set up my wpml-config.xml file according this article.

I was under the impression that Redux was meant to be compatible with WPML - so I just wanted to know why the WPML plugin was not working correctly on the theme options page. Is this a known issue? Am I doing something wrong? Is this known not to work? I just wanted some more info on the situation really. I'm certainly not expecting you create compatibility if this was never implemented in the first place.

As for financial support, I have already thought about pledging and probably will regardless of WPML integration. Redux is a great framework and I've got a ton of use out of it already.
